src/templates/admin_panel.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h1>DC Paint Drying</h1>
+<a href="..">Main Page</a>
```

Comment:
In `index.html` you use `<a href="{{ url_for('admin_panel') }}">`. Could a similar method be used here, instead of the ".."?